go/page.templ

```diff
@@ -92,3 +96,51 @@ templ PageTitle() {
 		{ children... }
 	</h1>
 }
+
+templ hamburgerButton() {
+	@templ.JSONScript("querySelectors", struct {
+		Button  string
+		Sidebar string
+		Page    string
+	}{
+		Button:  "." + classes.PageTitleWrapper + " > button",
+		Sidebar: "." + classes.Sidebar,
+		Page:    "." + classes.Page,
+	})
+	@Button(props.Button{}) {
+		@icons.Menu()
+	}
+	@hamburgerHandle.Once() {
```

Comment:
Did you already consider putting the JS into a file? We should avoid forcing the user to include frameworks, but it'd be reasonable in my opinion to include a versioned JS file for the popui alongside the styles.